doc/user/csv-annotations.md

```diff
@@ -0,0 +1,147 @@
+# Code Annotations for Cluster Service Versions
+
+## Overview
+
+This document describes the semantics of Cluster Service Version (CSV) [code annotations][code_annotations_design] and lists all possible annotations.
+
+## Usage
+
+All annotations have a `+operator-sdk:gen-csv:` prefix, denoting that they're parsed while executing [`operator-sdk olm-catalog gen-csv`][sdk_cli_ref].
+
+### Paths
+
+Paths are dot-separated string hierarchies with the above prefix that map to CSV [`spec`][csv_spec] field names.
+
+Example: `+operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Pod Count"`
+
+#### customresourcedefinitions
+
+- `customresourcedefinitions`: child path token
+	-	`displayName`: string
+	- `resources`: string, in the format `"kind,version,\"name\""`, where `kind`, `version`, and `name` are fields in each CSV `resources` entry
```

Comment:
For `name`, are quotation marks required all the time, or only when `name` contains whitespace?

Also, are the outer quotation marks and inner escapes necessary? Could the format be `kind,version,"name with spaces and a comma, maybe"`

Reply:
I should note in this doc that all strings are quoted strings, ex. `operator-sdk:cmd:path="value"` for all `string` types. The escapes are necessary when unquoting values. I'd rather use quotes and escapes like they are in a go source file. This makes parsing easier and eliminates edge cases.